wasm-node/javascript/src/public-types.ts

```diff
@@ -177,6 +177,14 @@ export interface Chain {
      */
     nextJsonRpcResponse(): Promise<string>;
 
+    /**
+     * JSON-RPC responses or notifications to be generated.
+     *
+     * @throws {@link JsonRpcDisabledError} If the JSON-RPC system was disabled in the options of the chain.
+     * @throws {@link CrashError} If the background client has crashed.
+     */
```

Comment:
This needs to mention that it's an async iterable, repeat what `nextJsonRpcResponse` mentions, and explain that iteration stops if the chain is removed.

Reply:
Yes, this make sense, I've explicitly added a comment marking this as an async iterable.

I have also copied the comment over from `nextJsonRpcResponse`, except for the part explaining simultaneous calls, because I believe this is an implied behaviour of `AsyncIterableIterator`. 

i.e.

```ts
// It's implied that 2 iterations happen, therefor each will yield different response
Promise.all([
  chain.jsonRpcResponses.next(),
  chain.jsonRpcResponses.next(),
]);
```